van-gogh-online-webstore/src/components/Coluna-salesoverview/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import './style.css'
+
+
+const Coluna = ({ valores }) => {
+
+    return <>
+        <div className='col-container'>
+            <div className='col'>{valores[0]}</div>
```

Comment:
Não é um problema, mas poderiamos ter um for aqui pra que ficasse dinâmico caso precisassemos de mais valores (mas se quiser pode deixar assim)

Reply:
feito